src/components/enterprise-banner/EnterpriseBanner.jsx

```diff
@@ -1,17 +1,32 @@
 import React, { useContext } from 'react';
+import { Link, useLocation } from 'react-router-dom';
 import { AppContext } from '@edx/frontend-platform/react';
-import { Container } from '@edx/paragon';
+import { Button, Container } from '@edx/paragon';
 
 import './styles/EnterpriseBanner.scss';
 
 export default function EnterpriseBanner() {
+  const location = useLocation();
   const { enterpriseConfig } = useContext(AppContext);
+  const isSearchPage = `/${ enterpriseConfig.slug }/search` === location.pathname;
   return (
     <div className="enterprise-banner bg-brand-secondary">
       <Container size="lg">
-        <h1 className="h2 mb-0 py-3 pl-3 border-brand-tertiary text-brand-secondary">
-          {enterpriseConfig.name}
-        </h1>
+        <div className="row banner-content">
+          <h1 className="h2 mb-0 py-3 pl-3 border-brand-tertiary text-brand-secondary">
+            {enterpriseConfig.name}
+          </h1>
+          {isSearchPage
+          && (
+            <Button
+              as={Link}
+              to={`/${ enterpriseConfig.slug }/skills-quiz`}
+              variant="inverse-primary"
```

Comment:
To make the button more prominent, can we try 'inverse-outline-primary' or 'inverse-outline-brand'?

Reply:
<img width="1432" alt="Screenshot 2021-09-14 at 1 20 26 PM" src="https://user-images.githubusercontent.com/11922730/133222441-1527e3b4-b560-4d4c-a641-48983f9c718d.png">

@moconnell1453 I tried them and with these two styles button will be prominent only when the user hovers on it.